src/test/ui/conflicting-impl-with-err.rs

```diff
@@ -0,0 +1,16 @@
+struct ErrorKind;
```

Comment:
Can we move the test so that it's not in the middle of `ui/` ?

Reply:
You mean a subdirectory or something like that?

Reply:
Yup, something like that :)

Reply:
I can, I suppose, move it to coherence, which seems like the best fit (though a poor one). I don't personally see much point though; I don't see much reason for the groupings of files within src/test/ui -- it seems like almost all or all of the time anyone will run all the tests, not a subset. But if you have a concrete suggestion I can implement it.

Reply:
coherence seems fine; the main point is to not have many files in the root of `ui/` cause that makes navigation ungreat.